vendor/void/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'gxi'
+pkgname=gxi
+version=git
+revision=1
+build_style=meson
+build_helper="rust"
+hostmakedepends="cargo pkg-config rust gettext git"
+makedepends="gtk+3-devel rust-std gettext-devel"
+checkdepends="appstream-glib desktop-file-utils"
+short_desc="GTK frontend for the xi text editor, written in Rust"
+maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
+license="MIT"
+homepage="https://github.com/Cogitri/gxi"
+
+if [ "$CROSS_BUILD" ]; then
+	configure_args+="-Drust-target=${RUST_TARGET}"
+fi
+
+do_fetch() {
+	# We do git clone instead of adding master to the distfiles=
+	# because checksum is actually useless here.
+	git clone https://github.com/Cogitri/gxi.git ${pkgname}-${version}
```

Comment:
Hum, since we're using etheral building couldn't we move the already by Drone CI cloned git repo to the build dir here? Would spare us from having some logic for PRs and stuff.

Reply:
No, this template is meant to be dropped inside a void-packages tree that will be cloned by drone CI.

Reply:
Ah, I was talking about using the already cloned gxi repo here instead of cloning it again